cmake/modules/RootInstallDirs.cmake

```diff
@@ -72,6 +74,17 @@ if(NOT DEFINED CMAKE_INSTALL_SYSCONFDIR)
   endif()
 endif()
 
+if(NOT DEFINED CMAKE_INSTALL_PYROOTDIR)
```

Comment:
it runs only if the user does not specify it.
the user can specify a relative path or an absolute path: the abs one can be outside of cmake_install_prefix